docs/security/users-and-teams/auditing.md

```diff
@@ -45,14 +50,32 @@ We take great care to ensure the security and integrity of your audit logs, to m
 
 To grant a user access to audit logs you can make use of a built-in User Role that contains **EventView**. All project related user roles contain it. **EventView** can also be scoped to narrow down which audit information a user can see, for example, it can be restricted to specific Projects or Environments. Learn more about [managing users and teams](/docs/security/users-and-teams/index.md).
 
+Audit entries older than the configured retention period (defaults to 90 days, configurable up to 365 days) are archived and can be access via the overflow menu (three dots) in the top right corner of the audit log page ("Manage archived events").
+
 In **Octopus 2019.1** we removed **AuditView** in an effort to simplify permissions so only **EventView** is now required.
 
 ### Modifying and deleting audit logs is prevented
 
-Octopus actively prevents modifying or deleting audit logs via its API. That being said, a user with the appropriate permissions to the `Events` table in your Octopus SQL Database could modify or delete records in that table. If you are concerned about this kind of tampering you should configure the permissions to the `Events` table in your Octopus SQL Database appropriately.
+Octopus actively prevents modifying or deleting audit logs within the configured retention period via its API. That being said, a user with the appropriate permissions to the `Events` table in your Octopus SQL Database could modify or delete records in that table. If you are concerned about this kind of tampering you should configure the permissions to the `Events` table in your Octopus SQL Database appropriately.
+
+Entries older than the retention period can be deleted by users with the appropriate permissions (typically `Octopus Manager`). An audit log entry will be created each time an archived event file is deleted. You can prevent archived files from being deleted by setting the configured folder to be read-only at the filesystem level.
```

Comment:
How does this work in practice? By limiting a folder to be read-only to prevent deletion of files, would this not also prevent the ability of Octopus Server to write the files to the archive folder too?

Reply:
I see what you mean. 

I think what this is trying to say is that other users should have their access level set to read-only.

Maybe an alternative could be (replacing 'You can prevent archived files...'):

`Archived files are saved at a filesystem level. So any user that has the appropriate permissions could view or delete these files. If this is a concern, you should restrict the permissions to access the configured folder appropriately`

@irynasemenova, what do you think?

Reply:
> would this not also prevent the ability of Octopus Server to write the files to the archive folder too

Is this question on the User's permission? Or Octopus application permission? @harrisonmeister 

> `Archived files are saved at a filesystem level. So any user that has the appropriate permissions could view or delete these files. If this is a concern, you should restrict the permissions to access the configured folder appropriately`

Yeah, this does seem clearer @sburmanoctopus

Reply:
👍 I've updated the text 🙂

Reply:
The question was on the Octopus application's permissions as it wasn't immediately clear. The new text is :+1: